### ISSUE TITLE
flux:grid.row label not visible

### ISSUE THREAD

**Issue by beam2web** [FluidTYPO3/flux#1513]
The Rendering of 
```
<flux:grid.row
    name="myRowName"
    label="notTranslatedRowLabel">
```
shows in the backend preview no output for the label or the name. The Structure for the output is placed in the /typo3conf/ext/flux/Classes/View/PreviewView.php Line 55

It would be create if the template rendering would stored in the common folder structure for partials, templates and layouts like https://fluidtypo3.org/documentation/templating-manual/templating/creating-templates/file-structure.html

And the attributes like name/label  should be visible in the backend like 

```
<flux:grid.column
     name="myColumnName"
     label="notTranslatedColumnLabel"
```

**Comment by NamelessCoder** [comment]
Rows have no labels - but columns do!

There are two main reasons why you can define a label but it doesn't get rendered:

* The "grid" you define is supposed to be possible to consume by any feature, not just page columns. When used in other places, rows may have labels. Even if it doesn't get rendered, it can also serve some documentation purposes in the form definition and when debugging the grid.
* The "label" is a universal property for any component - though not all of them use the property. Most notable cases are `Grid` and `Row`.

So that is "by design".

I'm not quite sure what you mean by:

> It would be create if the template rendering would stored in the common folder structure for partials, templates and layouts like https://fluidtypo3.org/documentation/templating-manual/templating/creating-templates/file-structure.html

...which doesn't seem to be related to this issue?

**Comment by beam2web** [comment]
I think it would be better not to store the complete output structure in PHP files like /typo3conf/ext/flux/Classes/View/PreviewView.php for a better update and maintenance.

For the structures it would be better to use partials and templates and call the needed information from viewHelpers.  So the developers could easy overwrite extension partials or templates in there own extensions. At the moment, it's difficult to implement missing things, and I need to override the whole class, and newer updates need to patch individually

**Comment by NamelessCoder** [comment]
I'm still not completely sure what you mean. If I understood it correctly that you meant the HTML markup that goes into rendering previews (namely grids) should be refactored to allow overrides, the opposite has happened: a more vanilla version of the grid rendering is now used, much closer to how the core does it natively. So there should no longer be a need to override PHP files...

Please take a look at the https://github.com/FluidTYPO3/flux/pull/1563 pull request, will be included in release 9.0 - but remove your override of PreviewView first.

If you test the state of the repository after this was merged, you should see column labels being rendered - but it is no longer possible for us to manipulate the resulting HTML markup. You should be able to use all the core's features concerning columns, and you can target them with CSS selectors if I'm not mistaken. The main cause for the headache is, I hope, gone.
